src/grass_gis_helpers/vector.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+#
+############################################################################
+#
+# MODULE:      vector.py
+# AUTHOR(S):   Anika Weinmann, Julia Haas
+#
+# PURPOSE:     functions for general vector processing
+# COPYRIGHT:   (C) 2024 by mundialis GmbH & Co. KG and the GRASS
+#              Development Team
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+############################################################################
+
+import grass.script as grass
+
+
+def patch_vector(vector_list, output):
+    """Patches vector data from a list
+
+    Args:
+        vector_list (list): list with vectors to patch
+        output (str): output map
+    """
+    # patch several vectors (e.g. from parallel imports)
+    if len(vector_list) > 1:
+        grass.run_command(
+            "v.patch",
+            input=vector_list,
```

Comment:
Wenn die Daten gepatcht werden, kann man sie danach ja eigentlich auch löschen. Man könnte evtl. noch `rm_vectors` als optionalem Parameter angeben. Und wenn es nicht None ist, dann kann man die `vector_list` dran hängen. Man kann natürlich auch davon ausgehen, dass der User das selber außerhalb der Funktion macht.

Reply:
habe ich in die Funktion verschoben